### DISCUSSION TITLE
Weblate permissions

### DISCUSSION THREAD

**Discussion by savornicesei** [WeblateOrg/weblate#6437] [Q&A]
Hi,

I wanted to add "Manage users" permission to Manager role but it does not seem possible. 
Looks like not all [privileges](https://docs.weblate.org/en/latest/admin/access.html#privileges) from documentation are available for creating new roles - is this by design?

Thank you.

![image](https://user-images.githubusercontent.com/917232/129947917-da0fe0b6-8061-4ffe-baa2-754cb305a3d1.png)

**Comment by nijel** [comment]
It's not possible to customize built-in roles, see end of https://docs.weblate.org/en/latest/admin/access.html#managing-per-project-access-control

**Reply by savornicesei** [reply]
The Aha! moment.

The "Manage user" permission does show up in the list when creating a new role. I've read the docs previously but I was confused not seeing the permission in the list for Administration.

Awesome.
Thank you!
